content/docs/releases/release-notes/release-notes-1.14.md

```diff
@@ -14,12 +14,25 @@ support for creating [CA certificates with "Name Constraints" and "Authority Inf
 >
 > Read [The cert-manager Command Line Tool (cmctl) page](../../reference/cmctl.md) to learn more.
 
+## `v1.14.2`
+
+> 📢 When upgrading to cert-manager release 1.14, please skip `v1.14.0` and `v1.14.1` and install this patch version instead.
+
+### Changes since `v1.14.1`
+
+#### Bug or Regression
+
+- BUGFIX: cert-manager CA and SelfSigned issuers incorrectly copied the critical flag from the CSR instead of re-calculating that field themselves. (https://github.com/cert-manager/cert-manager/pull/6727, https://github.com/jetstack-bot)
+- Helm: Fix a bug in the logic that differentiates between 0 and an empty value. (https://github.com/cert-manager/cert-manager/pull/6729, https://github.com/jetstack-bot)
```

Comment:
Need to be Markdown links. 
![image](https://github.com/cert-manager/website/assets/978965/13dd70c4-487a-4f1d-add4-d052a759e459)

Reply:
Sorry for doing this wrong a second time.